django-verdant/rca/context_processors.py

```diff
@@ -41,3 +42,9 @@ def global_vars(request):
         'SEO_NOINDEX': settings.SEO_NOINDEX,
         'RCA_LOGIN_DISABLED': settings.RCA_LOGIN_DISABLED,
     }
+
+
+def navigation_via_api(request):
```

Comment:
Could we add a check that we're not in the Wagtail admin, something like:

```
if settings.WAGTAIL_ADMIN_URL not in request.whatever:
    return {'things': ... }
```
(You might need to create this setting and refactor whats in `urls.py`)

Reply:
This isn't something I've tried yet, but have been meaning to :-)

Reply:
There doesn't seem to be a setting for that, well not that I can find in the docs.
This _works_

```[python]
    if not request.path.startswith('/admin/'):
        ...
```
But... if for some reason the admin path changes it will break, so it's not ideal

Reply:
You could create the settings and use it in `urls.py`, then if it did change it wouldn't break

Reply:
Added this in